.codecov.yml

```diff
@@ -0,0 +1,26 @@
+codecov:
```

Comment:
Do we really want this? I'm not the biggest fan of it due to the lack of accuracy

Reply:
this discussion needs to be brought to boilerplate.